Bullet Time/Assets/Scripts/TimeManager.cs

```diff
@@ -1,14 +1,20 @@
 using UnityEngine;
+using UnityEngine.Audio;
 
 public class TimeManager : MonoBehaviour {
 
 	public float slowdownFactor = 0.05f;
 	public float slowdownLength = 2f;
 
+    public AudioMixer MasterMixer;
+
 	void Update ()
 	{
 		Time.timeScale += (1f / slowdownLength) * Time.unscaledDeltaTime;
 		Time.timeScale = Mathf.Clamp(Time.timeScale, 0f, 1f);
+
+        float pitch = Mathf.Clamp(75 + (25 * Time.timeScale), 0, 100) / 100;
```

Comment:
Formatting kinda messed up due to tabs/spaces

Reply:
nice